File: src/main/java/com/thebluehats/server/game/utils/SortCustomEnchantByName.java
package com.thebluehats.server.game.utils;

import java.util.Comparator;

import com.thebluehats.server.game.managers.enchants.CustomEnchant;

public class SortCustomEnchantByName implements Comparator<CustomEnchant<?>> {
    @Override
    public int compare(CustomEnchant<?> a, CustomEnchant<?> b) {
        try {
            return a.getName().compareTo(b.getName());
        } catch (NullPointerException exception) {
            System.out.println("Error! Missing name information for an enchant.");
        }

        return 0;
    }
}

File: src/main/kotlin/com/thebluehats/server/game/utils/SortCustomEnchantByName.kt
package com.thebluehats.server.game.utils

import com.thebluehats.server.game.managers.enchants.CustomEnchant
import java.util.*

class SortCustomEnchantByName : Comparator<CustomEnchant> {
    override fun compare(a: CustomEnchant, b: CustomEnchant): Int {
        try {
            return a.name.compareTo(b.name)
        } catch (exception: NullPointerException) {
            println("Error! Missing name information for an enchant.")
        }
        return 0
    }
}